libs/payments/paypal/src/lib/paypalCustomer/paypalCustomer.factories.ts

```diff
@@ -0,0 +1,53 @@
+import { faker } from '@faker-js/faker';
+import {
+  CreatePaypalCustomer,
+  ResultPaypalCustomer,
+  UpdatePaypalCustomer,
+} from './paypalCustomer.types';
+
+export const ResultPaypalCustomerFactory = (
+  override?: Partial<ResultPaypalCustomer>
+): ResultPaypalCustomer => ({
+  ...override,
```

Comment:
Shouldn't the override's be at the end of object definition?

Reply:
Yep, updated!